data/map.txt

```diff
@@ -25960,7 +25960,7 @@ planet "New Kansas"
 
 planet "New Portland"
 	attributes "dirt belt" farming
-	landscape land/sky3
+	landscape land/mountain19-harro
```

Comment:
This replacement image doesn't have a reddish-looking sun, which is a key part of the planet's description.